### PR TITLE
fix(websocket): deprecation warning & 64-bit unsigned int body length

### PR DESCRIPTION
**Pull request by KhafraDev**

### CODE REVIEW

File: lib/websocket/frame.js [hunk 1]
```diff
@@ -45,7 +45,8 @@ class WebsocketFrameSend {
     if (payloadLength === 126) {
       new DataView(buffer.buffer).setUint16(2, bodyLength)
     } else if (payloadLength === 127) {
-      // TODO: optimize this once tests are added for payloads >= 2^16 bytes
+      // Clear extended payload length
+      buffer[2] = buffer[3] = 0
       buffer.writeUIntBE(bodyLength, 4, 6)
```
Comment (KhafraDev) [inline lib/websocket/frame.js:50]: I removed the todo comment because there likely isn't a faster way, unless I'm missing something obvious.

Comment (ronag) [reply lib/websocket/frame.js:50]: Assert body length fits in 6 bytes?

Comment (ronag) [reply lib/websocket/frame.js:50]: that's why it's an assertion?

Comment (KhafraDev) [reply lib/websocket/frame.js:50]: I don't think it's possible to allocate a buffer/typedarray/string that long.

Comment (KhafraDev) [reply lib/websocket/frame.js:50]: Although we should add an assertion on the receiving side, to ensure that we don't get a malicious frame.

**Pull request merged**